--- a/src/protobuf2json.c	2016-12-01 17:21:21.000000000 +0300
+++ b/src/protobuf2json.c	2019-12-26 03:38:43.753785489 +0300
@@ -201,7 +201,7 @@
     const void *protobuf_value = ((const char *)protobuf_message) + field_descriptor->offset;
     const void *protobuf_value_quantifier = ((const char *)protobuf_message) + field_descriptor->quantifier_offset;
 
-    if (field_descriptor->label == PROTOBUF_C_LABEL_REQUIRED) {
+    if (field_descriptor->label == PROTOBUF_C_LABEL_REQUIRED || field_descriptor->label == PROTOBUF_C_LABEL_NONE) {
       json_value = NULL;
 
       int result = protobuf2json_process_field(field_descriptor, protobuf_value, &json_value, error_string, error_size);
